conf/machine/qemuriscv64.conf

```diff
@@ -15,8 +15,6 @@ MACHINE_FEATURES = "screen keyboard ext2 ext3 serial"
 
 KERNEL_IMAGETYPE = "vmlinux"
 
-GDBVERSION = "riscv"
```

Comment:
this would make it use gdb from oe-core, we dont want that yet

Reply:
Argh, you are right. I jumped the gun here.